frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -172,3 +172,31 @@ Feature: Create Application from git form
                   | https://github.com/sclorg/django-ex.git        | django-ex-git-app  | django-ex-git         |
                   | https://github.com/spring-projects/spring-boot | spring-bottom-app  | openshift-quickstarts |
                   | https://github.com/sclorg/nodejs-ex.git        | nodejs-ex-git-app  | nodejs-ex-git         |
+
+
+        @regression @manual
+        Scenario Outline: Dotnet Builder iamge detection for git url "<git_url>": A-06-TC13
+            Given user is at Import from git page
+             When user enters Git Repo url as "<git_url>"
+              And user selects Show advanced Git options
+              And user enters Context dir as "<dir_name>"
+             Then git url gets Validated
+              And builder image is detected
+
+        Examples:
+                  | git_url                                                   | dir_name | app_name              | name              |
+                  | https://github.com/redhat-developer/s2i-dotnetcore-ex.git | /app     | dotnetcore-ex-git-app | dotnetcore-ex-git |
+
+
+        @regression @manual
+        Scenario Outline: Select Builder iamge manually for server related git urls": A-06-TC14
+            Given user is at Import from git page
+             When user enters Git Repo url as "<git_url>"
+              And user selects "<builder_image>" builder image
+             Then git url gets Validated
+              And warning message displays as "Unabel to detect the Builder Image" in Builder section
```

Comment:
fix typo and order of the events.

```suggestion
              And git url gets Validated
              And warning message displays as "unable to detect the Builder Image" in Builder section
              Then user selects "<builder_image>" builder image
```

Reply:
Action statement should be performed on When statement
verification should be performed on Then statement

Reply:
For more clarity, removing the "Selecting the builder image" when statement